src/diffusers/pipelines/pag/pag_utils.py

```diff
@@ -25,123 +30,60 @@
 
 
 class PAGMixin:
-    r"""Mixin class for PAG."""
-
-    @staticmethod
-    def _check_input_pag_applied_layer(layer):
```

Comment:
where did this check go?

Reply:
why do we need this check any more btw? since the matching is now regex based, a user of the library could do a wide variety of things that would be near to impossible to validate. how could we verify this?

Reply:
IIUC, the regex itself should error out then right if there's no matching pattern? If we're not doing it already, could we add one?

Reply:
We do raise an error if no matches are found via regex, so that should be covered